src/main/java/com/example/benefit/controller/CalculationMethodController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.benefit.controller;
+
+import com.example.benefit.model.CalculationMethodDTO;
+import com.example.benefit.service.CalculationMethodService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class CalculationMethodController {
```

Comment:
When all endpoints in a controller have the same mapping like `/calculationMethod` we can move it to controller level with annotation `@RequestMapping` and remove from each specific endpoint